services/segment/projectTracker.ts

```diff
@@ -4,15 +4,26 @@ import { getAnalytics } from '../../analytics';
 
 const analytics = getAnalytics();
 
+export enum SegmentEvents {
```

Comment:
@CarlosQ96  It's not related to this issue, but I thought maybe it's good to use enum instead of string :)))

Reply:
Good idea.